xonsh/completers/bash_completion.py

```diff
@@ -16,6 +16,8 @@
 
 __version__ = "0.2.6"
 
+import typing as tp
```

Comment:
This should be grouped with other standard library imports

Reply:
OK. I will update. Maybe using isort will do this automatically. Will accept adding that too to this PR?

Reply:
I kind of dislike isort, so I would prefer if it wasn't added

Reply:
now it has black support though. Anyways I will update this manually.

Reply:
Cool thanks

Reply:
done